### PR TITLE
Enable the pod garbage collector

### PR DESCRIPTION
**Pull request by smarterclayton**
Referenced from #5107

@derekwaynecarr review

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
LGTM [merge]

File: Godeps/_workspace/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ func NewCMServer() *CMServer {
 		RegisterRetryCount:                10,
 		PodEvictionTimeout:                5 * time.Minute,
 		ClusterName:                       "kubernetes",
+		TerminatedPodGCThreshold:          12500,
```
Comment (derekwaynecarr) [inline Godeps/_workspace/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go]: Is this configurable?

Comment (smarterclayton) [inline Godeps/_workspace/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go]: Yes, via the extended master controller flags.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[test]

On Oct 30, 2015, at 6:41 PM, OpenShift Bot <notifications@github.com> wrote:

continuous-integration/openshift-jenkins/test FAILURE (
https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin/6503/)

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5540#issuecomment-152667552>.

**Comment by smarterclayton** [comment]
[test]

On Oct 30, 2015, at 8:41 PM, OpenShift Bot <notifications@github.com> wrote:

continuous-integration/openshift-jenkins/merge FAILURE (
https://ci.openshift.redhat.com/jenkins/job/merge_pull_requests_origin/3871/
)

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5540#issuecomment-152680476>.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
This is definitely breaking namespace deletion (either by blocking or by speed).

**Comment by smarterclayton** [comment]
This is going into a hotloop, no time to debug.  Punted from 1.1

**Comment by ncdc** [comment]
I just pulled this locally. If you create a project, then try to delete it, it is stuck Terminating and you see the log message `namespace_controller.go:262] Syncing namespace <whatever>` endlessly.

**Comment by smarterclayton** [comment]
Ah, this snuck back in the queue

**Pull request closed without merging**